docs/source/technical_tutorials.rst

```diff
@@ -3,6 +3,8 @@ Technical Tutorials
 
 This section of the documentation includes hands-on tutorials with example algorithms and data-sets. It is focused on learning how to use the MAAP platform, and not science-oriented.
 
+**Note:** Tutorials are written in Python *except* for those under :ref:`Working with R <working_with_r_section>`. For current documents written in R, please see :ref:`Working with R <working_with_r_section>`.
```

Comment:
Love the note at the top-level to provide more guidance. I added a link to the working with R section so a user can directly jump to the relevant content in this case!